types/V1.mli

```diff
@@ -48,10 +48,23 @@ open Result
 
     {e Release %%VERSION%% } *)
 
-(** {1 Abstract devices}
+type 'a pp = Format.formatter -> 'a -> unit
```

Comment:
I do appreciate mirage-types not having any dependency (since we require 4.03+, result must be present in any case)